src/single-spa-angular1.js

```diff
@@ -33,23 +31,23 @@ export default function singleSpaAngular1(userOpts) {
 	}
 
 	return {
-		bootstrap,
-		mount,
-		unmount,
+		bootstrap: bootstrap.bind(null, opts),
+		mount: mount.bind(null, opts),
+		unmount: unmount.bind(null, opts),
 	};
 }
 
-function bootstrap() {
+function bootstrap(opts) {
```

Comment:
Yeah I figured that I'd stick the pattern of each lifecycle function having opts available to it for consistency across all the lifecycle functions.